### PR TITLE
Submitting lip-7

### PR DESCRIPTION
**Pull request by viktorbunin**
Lip-7 is a process lip on the lifecycle of technical working groups.

### CODE REVIEW

**Review by dahliamalkhi — APPROVED** [review]
Thanks for starting this !

I am approving the initial draft. However, I have some comments towards future revisions:
1. I am not sure we want hard requirements like "minimum 3 members" (how do you enforce this, take attendance?..), "push reports every 2 weeks" (some TWGs may be dormant at periods), etc. I'd prefer to think of these as guidelines for a successful/effective TWG.

2. I would add option to include participants who are not members. for example, Node Operations WG includes 3rd party operators, others groups may include experts, merchant representation, etc.

3. TWG may have more than a single LIP as output. For example, Off-chain Protocols is on-going, already produced 3 LIPs.

File: lips/lip-7.md
Comment (kphfb) [inline lips/lip-7.md:17]: Do we want to close this off to only association members? I could see an argument for and against this.  Just curious what the reasoning was that you went with this model

File: lips/lip-7.md
Comment (kphfb) [inline lips/lip-7.md:27]: Why does the facilitator choose who will be part of the working group?  Is that to keep the groups small?  The reason I ask is that it could be odd if for example Bison Trails wants to join a working group but the facilitator says that they weren't chosen and thus can't participate in the group

File: lips/lip-7.md
Comment (kphfb) [inline lips/lip-7.md:34]: I'd echo Dahlia's comment that this should be a suggested practice rather than "you must post every 2 weeks"

File: lips/lip-7.md
Comment (kphfb) [inline lips/lip-7.md:36]: The timeline one is a bit strange to me.  If a working group isn't making progress quickly enough, now they need to be re-approved by the TSC to continue their work?

File: lips/lip-7.md
Comment (kphfb) [inline lips/lip-7.md:40]: "After the editing process, the TSC must approve the output before it can be circulated within the Association" - it's unclear what is being suggested here.  By creating a draft PR, are we not effectively "circulating within the Association"?

File: lips/lip-7.md
Comment (kphfb) [inline lips/lip-7.md:44]: Do we want to call out that a LIP manager can dedicate eng resources?  Doesn't that hold true to anyone (even if not involved in the TWG)?

File: lips/lip-7.md
Comment (kphfb) [inline lips/lip-7.md:49]: > at which point it becomes up to the maintainers to implement it

I'm not sure that it becomes up to the maintainers to implement it.  Maintainers may not have the bandwidth to fully implement it and may require additional resources from other Association members

File: lips/lip-7.md
Comment (viktorbunin) [reply lips/lip-7.md:46]: Updating, thanks! (to Diem)

File: lips/lip-7.md
Comment (viktorbunin) [reply lips/lip-7.md:17]: This is a great point. I was thinking originally that it would just be association members, but I don't see a truly stellar reason to restrict it. If for some reason there is a problem with having it be open, it can always be restricted in the future. Updating this!

File: lips/lip-7.md
Comment (viktorbunin) [reply lips/lip-7.md:27]: This is a great point. The intention here was to have the facilitator own drumming up support and interest, but not to be exclusionary. I will update to make it clear that membership is open to all. Thank you for pointing it out!

File: lips/lip-7.md
Comment (viktorbunin) [reply lips/lip-7.md:34]: Updated, thank you!

File: lips/lip-7.md
Comment (viktorbunin) [reply lips/lip-7.md:36]: Fair point. I also removed expected output because that can also change over time. Really the only thing we need to watch out for is material changes in scope because we shouldn't have a TWG get approval to work on one thing and then pivot to something completely different without getting approval for the new direction.

File: lips/lip-7.md
Comment (viktorbunin) [reply lips/lip-7.md:44]: Sorry, but I'm not understanding your question. Can you say it a different way?

File: lips/lip-7.md
Comment (viktorbunin) [reply lips/lip-7.md:40]: Updated to majority. Also by circulating, I meant that it gets the stamp of approval from the TSC as a recommendation. I'll update to make that more clear.

**Pull request merged**
